Source/NSPopUpButtonCell.m

```diff
@@ -609,6 +609,7 @@ - (NSImage *) _currentArrowImage
 
       if (_pbcFlags.preferredEdge == NSMaxYEdge)
         {
+	  NSLog(@"\tDOWN ARROW!!!!!!");
```

Comment:
Looks like a leftover.

Reply:
Still debugging that's why I made it a draft.  This issue is really irking me. 🤨